source/adapters/level_zero/event.cpp

```diff
@@ -463,6 +465,70 @@ UR_APIEXPORT ur_result_t UR_APICALL urEventGetProfilingInfo(
 
   ze_kernel_timestamp_result_t tsResult;
 
+  // A Command-buffer consists of three command-lists for which only a single
+  // event is returned to users. The actual profiling information related to the
+  // command-buffer should therefore be extrated from graph events themsleves.
+  // The timestamps of these events are saved in a memory region attached to
+  // event usning CommandData field. The timings must therefore be recovered
+  // from this memory.
+  if (Event->CommandType == UR_COMMAND_COMMAND_BUFFER_ENQUEUE_EXP) {
+    if (Event->CommandData) {
+      struct command_buffer_profiling_t *ProfilingsPtr;
```

Comment:
Shouldn't need the `struct` prefix, that's a C convention rather than C++ (same for other places)
```suggestion
      command_buffer_profiling_t *ProfilingsPtr;
```